sync-core/src/main/java/com/cloudant/sync/sqlite/SQLDatabaseFactory.java

```diff
@@ -157,30 +158,7 @@ private static boolean validateOpenedDatabase(SQLDatabase db) {
         }
     }
 
-    /**
-     * <p>Update schema for give {@code SQLDatabase}</p>
-     *
-     * <p>Each input schema has a version, if the database's version is
-     * smaller in the schema version, the schema statements are executed.</p>
-     *
-     * <p>SQLDatabase's version is defined as:</p>
-     *
-     * <pre>    PRAGMA user_version;</pre>
-     *
-     * <p>In the schema statements, it must contains statement to update
-     * database version:</p>
-     *
-     * <pre>    PRAGMA user_version = version</pre>
-     *
-     *
-     * @param database
-     * @param schema
-     * @param version
-     * @throws SQLException
-     *
-     * @see com.cloudant.sync.sqlite.SQLDatabase#getVersion()
-     */
-    public static void updateSchema(SQLDatabase database, String[] schema, int version)
+    public static void updateSchema(SQLDatabase database, Migration migration, int version)
```

Comment:
Fixed in 8bda1601e9844f9930dfd48bd4f3b80d4c097fb8, where I also re-add the exception thrown if the migration fails.